src/js/components/Text/Text.js

```diff
@@ -2,15 +2,27 @@ import React, { forwardRef } from 'react';
 
 import { StyledText } from './StyledText';
 
-const Text = forwardRef(({ color, tag, as, a11yTitle, ...rest }, ref) => (
-  <StyledText
-    as={!as && tag ? tag : as}
-    colorProp={color}
-    aria-label={a11yTitle}
-    {...rest}
-    ref={ref}
-  />
-));
+const Text = forwardRef(
+  ({ color, tag, as, a11yTitle, truncate, children, ...rest }, ref) => {
+    const conditionalProps = {
+      truncate,
+      children,
+    };
+    if (truncate) {
+      conditionalProps.title = children;
```

Comment:
I think we would like to use the Tip component instead of the plain HTML title, this way we'll maintain the grommet's look & feel consistency.